src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.Icu.cs

```diff
@@ -8,7 +8,7 @@ namespace System.Globalization
 {
     public partial class CultureInfo : IFormatProvider
     {
-        internal static CultureInfo GetUserDefaultCulture()
+        internal static CultureInfo IcuGetUserDefaultCulture()
```

Comment:
>IcuGetUserDefaultCulture [](start = 36, length = 24)

we need to double check ICU is going to return the correct default locale. I expect so but we need to double check it.

Reply:
I'll double check this later. Thanks.